resources/views/welcome.blade.php

```diff
@@ -69,7 +69,7 @@
                 <div class="ris-title">Aktuelle Themen</div>
 
                 <a class="ris-card-list__item" title="Bewohnerparken Köln-Lindenthal"
```

Comment:
on line 96 and 120 i see the same code markup, could it be moved in a separate component?